wordpress-mobile/WordPress-iOS
# Activity Log > no activities: update text

Fixes #10130 

To test:
- On a site with no activity, go to Site > Activity.
- Verify the text is now:

<img width="350" alt="al_text" src="https://user-images.githubusercontent.com/1816888/45247042-0dd0fe00-b2c2-11e8-92de-1967bc7f7d85.png">
